src/site/content/en/blog/interop-2022/index.md

```diff
@@ -0,0 +1,142 @@
+
+For example, the following three card components have a header and footer aligned with the adjacent card headers and footers, even though each card has an independent grid. This pattern works because each card is an item that spans three rows of the parent grid, then uses subgrid to inherit those rows into the card.
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/QIrr5x5M7qCIo8rHMpOC.png", alt="A picture of a three card component where the headers and footers are aligned between the cards", width="800", height="205" %}
+
+Codepen: [https://codepen.io/rachelandrew/pen/yLzgowp](https://codepen.io/rachelandrew/pen/yLzgowp)
+
+### Also included
+
+- CSS Containment (the `contain` property)
+- The `<dialog>` element
+- Form controls
+- Typography and Encodings: including `font-variant-alternates`, `font-variant-position`,  the `ic` unit, and CJK text encodings
+- Web Compat, which focuses on differences between browsers that have caused site compatibility issues affecting end users
+
+The following areas made great progress through the Compat 2021 project, but there is still room for improvement. Therefore, they have been included in Interop 2022, in order that the remaining issues can be addressed.
+
+- Aspect Ratio
+- Flexbox
+- Grid
+- Sticky Positioning
+- Transforms
+
+
+### Investigation efforts
+
+In addition to the 15 focus areas, Interop 2022 includes three investigation efforts. These are areas that are problematic and need improvement, but where the current state of the specifications or tests aren’t yet good enough to be able to score progress using test results:
+
+- Editing, contenteditable and execCommand
+- Pointer and Mouse Events
+- Viewport Measurement
+
+Browser vendors and other stakeholders will collaborate on improving the tests and specifications for these areas, so that they could be included in future iterations of this effort.
+
+
+## Measuring success and tracking progress
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/YSLThucidlfsbtruHrkK.png", alt="An image showing scores per browser", width="800", height="366" %}
+
+The existing [web-platform-tests dashboard](https://wpt.fyi/) will be used to track the progress in the 15 focus areas. For each area, a set of tests have been identified. Browsers are then scored against these tests, giving a score for each area and an overall score for all 15 areas.
+
+To follow along and track the progress, check out the [Interop 2022 dashboard](https://wpt.fyi/interop-2022). Over the course of the year, you can follow along and see how the platform you build for is improving. 
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/FWFA2iKpQqJc3KpEHIij.png", alt="An image of a table with scores for many areas for all major web browsers", width="800", height="530" %}
```

Comment:
What is this table? It doesn't say what it is, where it has come from. If there is a link to the actual table then add it as a caption. Or, if the data is important use an actual table. Right now it doesn't have any context.

Reply:
Added a caption. Data is not the important, but the visual is, to get a feel for what the dashboard looks like.